client/src/components/embedding/index.tsx

```diff
@@ -24,12 +25,20 @@ import { AnnoMatrixObsCrossfilter } from "../../annoMatrix";
 import { getFeatureFlag } from "../../util/featureFlags/featureFlags";
 import { FEATURES } from "../../util/featureFlags/features";
 import { sidePanelAttributeNameChange } from "../graph/util";
+import { shouldShowOpenseadragon } from "../../common/selectors";
+import { ImageToggleWrapper, ImageDropdownButton } from "./style";
+import Opacities from "./components/Opacities";
 
 interface StateProps {
   layoutChoice: RootState["layoutChoice"];
   schema?: Schema;
   crossfilter: RootState["obsCrossfilter"];
   sideIsOpen: RootState["panelEmbedding"]["open"];
+  // eslint-disable-next-line react/no-unused-prop-types -- used in shouldShowOpenseadragon
+  config: RootState["config"];
+  // eslint-disable-next-line react/no-unused-prop-types -- used in shouldShowOpenseadragon
+  panelEmbedding: RootState["panelEmbedding"];
+  imageUnderlay: RootState["controls"]["imageUnderlay"];
```

Comment:
Embedding component now needs these props to determine whether to show the Image toggle button and its dropdown or not